dist/index.js

```diff
@@ -732,6 +740,7 @@ const create3DContrib = (svg, userInfo, x, y, width, height, settings, isForcedA
         const week = Math.floor(diffDate(startTime, cal.date.getTime()) / 7);
         const baseX = offsetX + (week - dayOfWeek) * dx;
         const baseY = offsetY + (week + dayOfWeek) * dy;
+        // ref. https://github.com/yoshi389111/github-profile-3d-contrib/issues/27
```

Comment:
Please don't ask me how this occurred! I suspect my IDE playing a trick on me here! Sorry!